src/libAtomVM/CMakeLists.txt

```diff
@@ -140,23 +155,16 @@ else()
     set(ATOMVM_VERSION ${ATOMVM_BASE_VERSION})
 endif()
 
+target_include_directories(libAtomVM PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Let's add a comment here (I should have done this before), saying that this is needed to pick up the generated `version.h` file.